runtime/doc/options.txt

```diff
@@ -7170,6 +7173,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     *'ttyfast'* *'tf'* *'nottyfast'* *'notf'*
 'ttyfast' 'tf'		Removed. {Nvim}
 
+	Support for slow terminal connections has been removed.  "set tf" is
```

Comment:
There should not be an empty line above this, to be consistent with the other entries.